nmoscommon/webapi.py

```diff
@@ -382,16 +422,18 @@ def annotate_function(func):
         return func
     return annotate_function
 
-def errorhandler(*args,**kwargs):
+
+def errorhandler(*args, **kwargs):
     def annotate_function(func):
         func.errorhandler_args = args
         func.errorhandler_kwargs = kwargs
         return func
     return annotate_function
 
+
 def on_json(path):
     def annotate_function(func):
-        func.sockets_on = path
+        func.socket_path = path
```

Comment:
Is there any danger of this being used accessed of this file and constituting a breaking change, or is it only internal?

Reply:
should only be local to the web API. It is both set by the web API in the decorator, and then found (`getattr(routesFunction, "socket_path)`) by the web API also. After a quick search the only place it seems to also be used is in rd-ips-python-ipppython , but thats just a copy of the webapi.py